site/content/en/references/kustomize/cmd/create/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "kustomize create"
+linkTitle: "create"
+type: docs
+weight: 4
+description: >
+    Create a new kustomization in the current directory
+---
+
+The `kustomize create` command will create a new kustomization in the current directory.
+
+When run without any flags the command will create an empty `kustomization.yaml` file that can then be updated manually or with the `kustomize edit` sub-commands.
+
+Example command:
+
+```
+kustomize create --namespace=myapp --resources=deployment.yaml,service.yaml --labels=app:myapp
+```
+
+Output from example command:
+
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- deployment.yaml
+- service.yaml
+namespace: myapp
+commonLabels:
+  app: myapp
+```
+
+### Detecting resources
+
+> NOTE: Resource detection will not follow symlinks.
```

Comment:
Would you say this is still valid? Running kustomize create --autodetect with a file that is a symlink detects the file and adds it to kustomization.yaml, but of course then kustomize build fails because the referenced file is not in the root directory, not sure if this is what this note was originally referring to

Reply:
I'm not sure either. Let's remove this section.